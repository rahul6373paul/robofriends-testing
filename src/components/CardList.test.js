import { shallow, mount, render,configure } from 'enzyme';
import React from 'react';
import { CardList } from './CardList';
import Adapter from 'enzyme-adapter-react-16';  

configure({ adapter: new Adapter() });

it('expect to render CardList Component', () => {

    const mockRobots = [
        {
            id: 1,
            name: 'John Doe',
            email: 'doe@gmail.com',
            username: 'JohnJohn'
        }
    ]

    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot()    
});

