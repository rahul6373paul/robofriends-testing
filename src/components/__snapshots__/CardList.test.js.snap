// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList Component 1`] = `
<div>
  <Card
    email="doe@gmail.com"
    id={1}
    key="0"
    name="John Doe"
  />
</div>
`;
